Sources/KituraNet/ClientRequest.swift

```diff
@@ -269,12 +269,12 @@ public class ClientRequest {
         }
         self.path = fullPath
 
-        if let username = url.user {
```

Comment:
We must retain this code. The username and password will continue to be accessible using the `userName` and `password` properties. They only won't be included in the `url` property.